packages/election/graphql/resolvers/_mutations/cancelCandidacy.js

```diff
@@ -0,0 +1,15 @@
+const getElections = require('../../../lib/getElections')
+
+const {
+  Roles,
+  ensureSignedIn
+} = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, { slug }, { pgdb, user: me, req }) => {
```

Comment:
Assuming this mutation is for admin users to cancel any users candidacy, this appears to lack some bussiness logic. (As it is, this mutation would cancel all candidacies of the admin user using this mutation.)

A flow like this might be working
1. Require mutation to accept a User ID besides `slug` (usually named `userId`).
2. Query `slug` in elections. Throw error if `slug` can't be found. If found, keep election ID.
3. Query `userId` in users. Throw error if user can't be found. If found, keep user ID.
4. Query `electionCandidacies` for `slug` and `userId`. If nothing is to be found, throw an error.
5. If election candidacy was to be found, delete it.